Week-2/Article/friends.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>F.R.I.E.N.D.S</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <h1>F <font color=red>.</font>R
```

Comment:
There is a mix of constant colors and hex values. Can we please use one format for colors? Also they can come as css classes too instead of inline styles.